robot_skills/src/robot_skills/robot.py

```diff
@@ -184,6 +185,10 @@ def seq_or_none(obj):
 
             # Grippers
             matching_grippers = set()
+            # ToDO: HACK for not specifying any requirements
```

Comment:
+100 for this. Makes API much easier to use

Reply:
Except you can't have a guarantee that you have sufficient capabilities if you don't specify what you need to have.

Reply:
I know, but It would be too much to change at the moment.